components/staging/staging.js

```diff
@@ -263,6 +266,10 @@ StagingViewModel.prototype.invalidateFilesDiffs = function() {
     file.diff().invalidateDiff();
   });
 }
+StagingViewModel.prototype.cancelAmendEmpty = function() {
```

Comment:
Instead of `cancelAmendEmpty()` is a literally a proxy function for `resetMessages()`.  

Should we just call `resetMessages()` directly?